docker/production.yml

```diff
@@ -238,8 +238,8 @@ services:
       - 9000
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-      - ../../config:/data
-      - ../../config/portainer:/data/portainer
+      - ../../portainer:/data
```

Comment:
In the rest of this file you use `elastic-data` and `redis-data` so is this `portainer-data`?

Happy either way.

Reply:
hmm, I didn't think of that. For `redis` and `ES` it makes sense since they're databases, but the `portainer` directory is not storing data, just config files, but I'm ok adding `data` for consistency.